openmetadata-ui/src/main/resources/ui/playwright/e2e/Flow/PersonaFlow.spec.ts

```diff
@@ -192,6 +192,40 @@ test.describe.serial('Persona operations', () => {
     );
   });
 
+  test('Set and remove default persona should work properly', async ({
+    page,
+  }) => {
+    await page
```

Comment:
test i would like. 

Create normal user and check landing page
Admin creates persona and the user into it, add some widgets to that persona
User logs in checks the widgets on landing page,
Admin updates  the persona and removes the default.
User logs in again and sees the landing page without those assets

Reply:
Added.